examples/hybrid_jobs/1_Quantum_machine_learning_in_Amazon_Braket_Hybrid_Jobs/Quantum_machine_learning_in_Amazon_Braket_Hybrid_Jobs.ipynb

```diff
@@ -60,25 +60,26 @@
      "name": "stdout",
```

Comment:
<p><code>    The number of layers determines how many rotation angles are in the quantum circuit.  </code></p><p>Why is this formatted as a code block?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>sorry, i don't see it as a code block on my end. </p>